components/controlnet_aux_processor/fondant_component.yaml

```diff
@@ -0,0 +1,20 @@
+name: Controlnet aux processor
+description: Component that processes image with controlnet aux processors
+image: ghcr.io/ml6team/controlnet_aux_processor:latest
+
+consumes:
+  images:
+    fields:
+      data:
+        type: binary
+
+produces:
+  conditioning:
```

Comment:
are there potential use cases where we might need to have more than one conditioning in the dataset (e.g. segmentation + canny, ...)? Or is that unlikely?

Reply:
Could happen, but I suppose the user just has to rewrite the component so it does multiple at once and writes them to different columns, or that it uses 2 separate components based on this one.